src/NotesLoaderBMS.cpp

```diff
@@ -1487,7 +1494,7 @@ void BMSSongLoader::AddToSong()
 		for( unsigned i = 0; i < loadedSteps.size(); i ++ )
 		{
 			Steps *steps = loadedSteps[i].steps;
-			steps->SetDifficulty( Difficulty_Medium );
+			//steps->SetDifficulty( Difficulty_Medium );
```

Comment:
This should have been removed because now the difficulty is set properly.  Commented out code that is left in usually means there might be a reason to want it again in the future.
Also, how does the new code handle BMSes that do not have a difficulty tag?